### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.11 es 4.78 (18 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#328] [Esercizi M-Z]
![es.78](https://github.com/user-attachments/assets/e09a43fb-ff83-4c6f-b164-d0ac1ed63de6)

**Comment by alem1105** [comment]
<img width="611" alt="image" src="https://github.com/user-attachments/assets/1063db9a-11c5-4128-be5d-0b67d2c474b6">
